lib/src/Base/Optim/MultiStart.cxx

```diff
@@ -0,0 +1,212 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief MultiStart optimization algorithm
+ *
+ *  Copyright 2005-2016 Airbus-EDF-IMACS-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include "openturns/MultiStart.hxx"
+#include "openturns/PersistentObjectFactory.hxx"
+#include "openturns/Cobyla.hxx"
+#include "openturns/SpecFunc.hxx"
+#include "openturns/MonteCarloExperiment.hxx"
+#include "openturns/ComposedDistribution.hxx"
+#include "openturns/Uniform.hxx"
+
```

Comment:
You should remove references to all the objects leaving into the Uncertainty layer (MonteCarloExperiment, ComposedDistribution, Uniform)